Emby.Server.Implementations/Session/WebSocketController.cs

```diff
@@ -88,6 +88,18 @@ private void OnConnectionClosed(object? sender, EventArgs e)
                 cancellationToken);
         }
 
+        /// <summary>
+        /// Gracefully closes all web sockets.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
```

Comment:
```suggestion
        /// <inheritdoc />
```

Reply:
Made the changes. Should've looked into what inheritdoc did! Thanks for taking a look at my stuff